pyk/src/pyk/__main__.py

```diff
@@ -389,6 +395,65 @@ def exec_json_to_kore(options: JsonToKoreOptions) -> None:
     sys.stdout.write('\n')
 
 
+def exec_parse_outer(options: ParseOuterOptions) -> None:
+    # TODO Desugar K language syntax to KAST compatible items
+    #      Import the prelude and add --no-prelude option
+    #      Trim away modules unreachable from the main/syntax modules (modulo some builtins)
+    #      Checks:
+    #        - Circular imports
+    #        - Modules with duplicate names
+    main_file_dir = Path(options.main_file.name).resolve().parent
```

Comment:
Do you want to `check_file_path` this, or is the error from `.resolve()` good enough?

Reply:
`options.main_file` is checked by the argument parser, so before we get to this `.resolve()` it should error there:

https://github.com/runtimeverification/k/blob/7cbf5551b5aca2e7a19a231611c8ceeb08bc5f0a/pyk/src/pyk/__main__.py#L602

Reply:
Nice!